ConceptMap/ConceptMap-Genomics-primary-sample-state.json

```diff
@@ -0,0 +1,141 @@
+          "code": "DNA-stabiliser",
+          "display": "DNA Stabiliser",
+          "target": [
+            {
+              "code": "706052002",
+              "display": "Evacuated blood collection tube (physical object)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "edta",
+          "display": "EDTA",
+          "target": [
+            {
+              "code": "702282003",
+              "display": "Non-evacuated blood collection tube, EDTA",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "ffpe-curls/scrolls",
+          "display": "FFPE Curls/Scrolls",
+          "target": [
+            {
+              "equivalence": "unmatched"
+            }
+          ]
+        },
+        {
+          "code": "ffpe-slides",
+          "display": "FFPE Slides",
+          "target": [
+            {
+              "equivalence": "unmatched"
+            }
+          ]
+        },
+        {
+          "code": "fresh-cells",
+          "display": "Fresh Cells",
+          "target": [
+            {
+              "code": "706049005",
+              "display": "Blood tube (physical object)",
+              "equivalence": "subsumes"
+            }
+          ]
+        },
+        {
+          "code": "fresh-tissue",
+          "display": "Fresh Tissue",
+          "target": [
+            {
+              "code": "434464009",
+              "display": "Tissue cassette (physical object)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "lithium-heparin",
+          "display": "Lithium Heparin",
+          "target": [
+            {
+              "code": "702284002",
+              "display": "Non-evacuated blood collection tube, lithium heparin (physical object)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "other",
+          "display": "Other",
+          "target": [
+            {
+              "code": "260787004",
+              "display": "Physical object (physical object)",
+              "equivalence": "wider",
+              "comment": "Any other suitable device/equipment that is an alternative to concepts provided in this Conceptmap. This should be provided by free text."
+            }
+          ]
+        },
+        {
+          "code": "RNA-stabiliser",
+          "display": "RNA Stabiliser",
+          "target": [
+            {
+              "code": "702280006",
+              "display": "Evacuated blood collection tube, ribonucleic acid stabilizer (physical object)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "streck",
+          "display": "Streck",
+          "target": [
+            {
+              "code": "706052002",
+              "display": "Non-evacuated blood collection tube, EDTA",
```

Comment:
Streck is a company which provides multiple tube types, need to confirm what exactly is being referenced here